### PR TITLE
release-20.1: cli: make cockroach demo start in insecure mode by default

### PR DESCRIPTION
**Pull request by rohany**
Release note (cli change): This change makes cockroach demo
start in insecure mode by default.

### CODE REVIEW

File: pkg/cli/interactive_tests/test_demo.tcl [hunk 1]
```diff
@@ -10,6 +10,8 @@ eexpect "Welcome"
 eexpect "your changes to data stored in the demo session will not be saved!"
 # Inform the necessary URL.
 eexpect "Web UI: http:"
+# See the notice about insecure mode.
+eexpect "Cockroach demo is running in insecure mode."
```
Comment (knz) [inline pkg/cli/interactive_tests/test_demo.tcl:14]: can you add an expect for this message in the two cases of the test below ("check that demo insecure says hello properly") thanks

**Review by knz — APPROVED** [review]
nice. LGTM with nit

**Comment by knz** [comment]
that's ready

**Comment by awoods187** [comment]
thanks for making this change!

**Pull request merged**